api/rest/client/methods.go

```diff
@@ -138,7 +138,8 @@ func (c *defaultClient) Status(ci cid.Cid, local bool) (api.GlobalPinInfo, error
 // StatusAll gathers Status() for all tracked items. If a filter is
 // provided, only entries matching the given filter statuses
 // will be returned. A filter can be built by merging TrackerStatuses with
-// a bitwise OR operation (st1 | st2 | ...).
+// a bitwise OR operation (st1 | st2 | ...). A "0" filter value (or
+// api.TrackerStatusUndefined), means all.
```

Comment:
@hsanjuan This seems inappropriate to me.

Passing invalid filter(say `ipfs-cluster-ctl status --filter invalidFilter`) returns all pin statuses. Something like this should return an error say `invalid filter value` or we can give a warning saying that `filter value was invalid so we are showing all pin statuses`

Also the behaviour is inconsistent. On passing invalid filter to the API, it would through error with no status.